src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile2.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+
+// Given_StorageFile2 , as Given_StorageFile is used in PR#2407
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given_StorageFile2
+	{
+		
+		String _filename;
+		
+		[TestInitialize]
+		public void Init()
+		{
+			_filename = DateTime.Now.ToString("yyyyMMddHHmmss") + ".txt";
+		}
+
+		[TestCleanup]
+		public void Cleanup()
+		{
+		}
+
+		[TestMethod]
+		public async void When_DateCreated()
+		{
+			
+			var _folder = Windows.Storage.ApplicationData.Current.LocalFolder;
+			Assert.IsNotNull(_folder, "cannot get LocalFolder - error outside tested method");
+
+			Windows.Storage.StorageFile _file = null;
+			
+      		DateTimeOffset _dateBefore = DateTimeOffset.Now;
```

Comment:
Indentation

Reply:
i don't know why you see this as such - when I open file for editing (in github), indentation seems ok.

Reply:
This is how it looks like. It's most probably a tabs/spaces issue:

![image](https://user-images.githubusercontent.com/5839577/72348309-25837800-36a8-11ea-96cd-b3c99a33d822.png)